src/main/java/maksym/fedorenko/bookstore/controller/BookController.java

```diff
@@ -20,28 +23,45 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoints for mapping books")
 @RequiredArgsConstructor
 @RestController
 @RequestMapping("/api/books")
 public class BookController {
     private final BookService bookService;
 
+    @Operation(
+            summary = "Retrieve all books",
+            description = "Receiving all books with pagination and sorting by fields."
+    )
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    public List<BookDto> getAll(Pageable pageable) {
+        return bookService.findAll(pageable);
     }
 
+    @Operation(
+            summary = "Retrieve a book by id",
+            description = "Get a book object by specifying its id."
+    )
     @GetMapping("/{id}")
     public BookDto getBookById(@PathVariable @Positive Long id) {
```

Comment:
validation of @PathVariable,  @RequestParam etc. will not work untill you add @Validated annotation to your controller class. therefore this @Positive is not being checked


so pls add that annotation :)